resolwe/flow/executors/docker/__init__.py

```diff
@@ -30,15 +36,56 @@ def start(self):
             'container_image': self.requirements.get('image', settings.FLOW_EXECUTOR['CONTAINER_IMAGE']),
         }
 
+        # Set resource limits.
+        limits = []
+        if 'cores' in self.resources:
+            # Each core is equivalent to 1024 CPU shares. The default for Docker containers
+            # is 1024 shares (we don't need to explicitly set that).
+            limits.append('--cpu-shares={}'.format(int(self.resources['cores']) * 1024))
```

Comment:
It would be great if executor injects limits in process so tools can be run depending on resources that they have. I.e. instead of:

    cuffquant --num-threads '{{ threads }}'

you can run:
 
    cuffquant --num-threads '{{ limits.cores }}'